specification/authorization/resource-manager/Microsoft.Authorization/2017-10-01-preview/authorization-RACalls.json

```diff
@@ -0,0 +1,612 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "AuthorizationManagementClient",
+    "version": "2017-10-01-preview",
+    "description": "Role based access control provides you a way to apply granular level policy administration down to individual resources or resource groups. These operations enable you to manage role definitions and role assignments. A role definition describes the set of actions that can be performed on resources. A role assignment grants access to Azure Active Directory users."
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json",
+    "text/json"
```

Comment:
>"text/json" [](start = 4, length = 11)

According to the linter, ARM does not support consuming and producing `"test/json"`